packages/adapter-vercel/index.d.ts

```diff
@@ -4,6 +4,30 @@ type Options = {
 	edge?: boolean;
 	external?: string[];
 	split?: boolean;
+	platform?: VercelConfig;
+};
+
+type ImageFormat = 'image/avif' | 'image/webp';
+
+type RemotePattern = {
+	protocol?: 'http' | 'https';
+	hostname: string;
+	port?: string;
+	pathname?: string;
+};
+
+type ImagesConfig = {
```

Comment:
does vercel expose any types that we should leverage here rather than writing them ourselves?

Reply:
The only type definitions I've been able to find were just the type definitions from the build output api docs. None of the existing dependencies expose them, and there doesn't appear to be a new dependency we could add to get them.

Reply:
yeah, looks like that's right. I couldn't find them either